api/src/main/kotlin/com/gmtkgamejam/models/Skills.kt

```diff
@@ -0,0 +1,16 @@
+package com.gmtkgamejam.models
+
+enum class Skills(val bitwiseId: Int) {
+    `2D_ART`(1),
+    `3D_ART`(2),
```

Comment:
The ` around these seem a bit out of place. couldn't we just go for ART_2D and ART_3D?
It seems accessing these also has to happen with the backticks around it, which IntelliJ annoyingly does not place for you.
If I'm the only one that finds this annoying feel free to ignore this comment

Reply:
Amended, thanks 👍